terra-scm-1.rockspec

```diff
@@ -0,0 +1,34 @@
+package = "terra"
+version = "scm-1"
+
+source = {
+   url = "git://github.com/zdevito/terra.git",
+   branch = "master",
+}
+
+description = {
+   summary = "A low-level counterpart to Lua.",
+   homepage = "",
+   license = "MIT",
+}
+
+build = {
+   type = "command",
+   build_command = [[
+   make clean
+
+    make LUAJIT_LIB=$(LUA_LIBDIR)/libluajit-5.1.a \
```

Comment:
Great, so I use `make PREFIX=$PREFIX install` now, works nicely.

LUAJIT_PREFIX doesn't work for me because I set lua headers to install in `$LUAJIT_PREFIX/include`, and not `$LUAJIT_PREFIX/include/luajit-2.0/`. This is to make LuaJIT install identically to PUC Lua. 

Would be good to not hard-code the specific libs, but instead grep for a pattern, and then use the directory in which the desired file resides. This would prepare us for PUC Lua support.